src/siaskydev.test.ts

```diff
@@ -29,4 +31,48 @@ describe.skip("siasky.dev end to end", () => {
     const buf = Buffer.from(parts[1], "base64");
     expect(buf.toString("ascii")).toEqual("thisistext");
   });
+
+  it("support rev number update higher than 255", async () => {
+    // create a random user
+    const rand = randomNumber(0, 1e6);
+    const user = new User(`john.doe+${rand}@gmail.com`, "test1234");
+    console.log(`Uploading for userid ${user.id}`);
+
+    // verify there is no entry
+    let existing = null;
+    try {
+      existing = await promiseTimeout(client.lookupRegistry(user, fileID), 1e4);
+    } catch (error) {
+      // expected
+    }
+    expect(existing).toBeNull();
```

Comment:
use expect toThrow assertion https://jestjs.io/docs/en/expect

Reply:
I tried that but it doesn't work for async functions of which you want to assert the result value. I also don't really care about the error thrown, wouldn't you normally use `toThrow` when you want to assert what the error?

Reply:
I would use, that's not it but the following's not working
`expect(async () => {await promiseTimeout(client.lookupRegistry(user, fileID), 1e4);}).toThrow('..my error')`
and I much rather use the return value of that function in the assert.

Reply:
you can try a suggestion from https://github.com/facebook/jest/issues/5538
```javascript
await expect(asyncFunc()).rejects.toThrow();
```

Reply:
https://stackoverflow.com/a/47887098

Reply:
I tried, it does not work the way I want it to. I don't see why we would rewrite it in an awkward way only to avoid the `try-catch` in a test file. I looked it up and everything, it's not that I did not try, I just don't see the issue using `try-catch` here.

I much rather remove this entire file if it is cause for concern or something we do not want to merge per se. I only kept it in because it is useful to have for debugging purposes.